cetmix_tower_jet_mode/models/cx_tower_jet.py

```
@@ -0,0 +1,41 @@
+        "jet_template_id.action_ids.state_from_id",
+        "jet_template_id.action_ids.state_to_id",
+        "jet_template_id.action_ids.priority",
+        "active_mode_id",
+        "active_mode_id.action_ids",
+    )
+    def _compute_available_actions(self):
+        super()._compute_available_actions()
+        for rec in self:
+            if rec.active_mode_id and rec.active_mode_id.action_ids and rec.action_available_ids:
+                rec.action_available_ids = rec.action_available_ids.filtered(
+                    lambda a: a.id in rec.active_mode_id.action_ids.ids
```

Ruff (0.15.14)

[warning] Function definition does not bind loop variable `rec`

(B023)